torch/csrc/inductor/aoti_runtime/model_container.h

```diff
@@ -129,31 +128,31 @@ class AOTInductorModelContainer {
     if (this->num_models() == 0) {
       throw std::runtime_error("No available models in container!");
     }
-    return models_[0]->constant_name(idx);
+    return models_[0]->constant_name(static_cast<int64_t>(idx));
```

Comment:
This is a lot of casting, should constant_name just actually have a size_t type? Or atleast an overload for it?

Reply:
It can be, I will check that later